docs/supplychain-sbom.md

```diff
@@ -411,9 +416,13 @@ When doing business within the European Economic Area (EEA), has the duty to ens
 
 * See also [Owner](#owner)
 
-| Do | Field name                    | Required | Data type | CycloneDX (PRE-PROPOSAL; UNSUPPORTED)                          | SPDX | Required by                        |
+<!--
+SPDX 2.3 doesn't support the CE fields. SPDX 3.0 should be used at a future date
+-->
+
+| Do | Field name                    | Required | Data type | CycloneDX (PRE-PROPOSAL; UNSUPPORTED)                          | SPDX 2.3 | Required by                        |
 | -- | :---------------------------- | :------- | :-------- | :------------------------------------------------------------- | ---- | ---------------------------------- |
-| 🟥 | Owner Name (Manufacturer)     | Yes      | Text, URL | bom.metadata[supplier,manufacturer], bom.components[].supplier |      | CRA-AII(1), NTIA-SBOM, DE-TR.5.2.2 |
+| 🟥 | Owner Name (Manufacturer)     | Yes      | Text, URL | bom.metadata[supplier,manufacturer], bom.components[].supplier | creationInfo.creators[], packages[].originator, packages[].supplier | CRA-AII(1), NTIA-SBOM, DE-TR.5.2.2 |
```

Comment:
Could you say a little about the differences between these three fields? I'm wondering if there's a missing role that should be added for clarity.

Reply:
They are comparable to the CycloneDX fields already filled in.

The creators[] field is pretty overloaded (I'll add a note about that below)

The originator field is defined here
https://spdx.github.io/spdx-spec/v2.3-RC1/package-information/#7.6

And the supplier field defined here
https://spdx.github.io/spdx-spec/v2.3-RC1/package-information/#7.5

Reply:
@sjn is there anything else to work on here, or is it resolved?